FirebaseFunctionsSwift/Tests/Unit/FunctionsTests.swift

```diff
@@ -189,43 +190,37 @@ class FunctionsTests: XCTestCase {
 
   // MARK: - App Check integration
 
-  // - (void)testCallFunctionWhenAppCheckIsInstalledAndFACTokenSuccess {
-//  _appCheckFake.tokenResult = [[FIRAppCheckTokenResultFake alloc] initWithToken:@"valid_token"
-//                                                                          error:nil];
-//
-//  NSError *networkError = [NSError errorWithDomain:@"testCallFunctionWhenAppCheckIsInstalled"
-//                                              code:-1
-//                                          userInfo:nil];
-//
-//  XCTestExpectation *httpRequestExpectation =
-//      [self expectationWithDescription:@"HTTPRequestExpectation"];
-//  __weak __auto_type weakSelf = self;
-//  _fetcherService.testBlock = ^(GTMSessionFetcher *_Nonnull fetcherToTest,
-//                                GTMSessionFetcherTestResponse _Nonnull testResponse) {
-//    // __unused to avoid warning in Xcode 12+ in g3.
-//    __unused __auto_type self = weakSelf;
-//    [httpRequestExpectation fulfill];
-//
-//    NSString *appCheckTokenHeader =
-//        [fetcherToTest.request valueForHTTPHeaderField:@"X-Firebase-AppCheck"];
-//    XCTAssertEqualObjects(appCheckTokenHeader, @"valid_token");
-//
-//    testResponse(nil, nil, networkError);
-//  };
-//
-//  XCTestExpectation *completionExpectation =
-//      [self expectationWithDescription:@"completionExpectation"];
-//  [_functions callFunction:@"fake_func"
-//                withObject:nil
-//                   timeout:10
-//                completion:^(FIRHTTPSCallableResult *_Nullable result, NSError *_Nullable error) {
-//                  XCTAssertEqualObjects(error, networkError);
-//                  [completionExpectation fulfill];
-//                }];
-//
-//  [self waitForExpectations:@[ httpRequestExpectation, completionExpectation ] timeout:1.5];
-  // }
-//
+  func testCallFunctionWhenAppCheckIsInstalledAndFACTokenSuccess() {
+    appCheckFake.tokenResult = FIRAppCheckTokenResultFake(token: "valid_token", error: nil)
+
+    let networkError = NSError(
+      domain: "testCallFunctionWhenAppCheckIsInstalled",
+      code: -1,
+      userInfo: nil
+    )
+
+    let httpRequestExpectation = expectation(description: "HTTPRequestExpectation")
+    fetcherService.testBlock = { fetcherToTest, testResponse in
+      httpRequestExpectation.fulfill()
```

Comment:
Nit: I realize this is a direct port but I think the expectation fulfillment should go after everything in the block? Otherwise the `XCTAssertEqual` could not be run in time.

Reply:
Done in next PR after #9300